src/bench/block_assemble.cpp

```diff
@@ -41,7 +41,7 @@ static CTxIn MineBlock(const CScript& coinbase_scriptPubKey)
     auto block = PrepareBlock(coinbase_scriptPubKey);
 
     while (!CheckProofOfWork(block->GetHash(), block->nBits, Params().GetConsensus())) {
-        assert(++block->nNonce);
+        ++block->nNonce;
```

Comment:
The `->nNonce` access has no effect without the assert. I would be in favor of keeping the assert on the following line.

Reply:
`++block`?

Note that this is as: `assert(++(block->nNonce));`

Reply:
Thanks, yeah figured that out here: https://github.com/bitcoin/bitcoin/pull/13534#discussion_r200772537 :P